README.md

```diff
@@ -164,6 +164,49 @@ A macro such as {pageVariable.foobar} allows the user access the value of any pr
 | Undefined | Logs warning and returns empty string |
 | Other     | Logs warning and returns empty string |
 
+## Cue Metadata Tracks
+
+An optional feature that allows the manipulation of metadata tracks, specifically in the case of working with advertising cue points.
+
+For example, an ad integration may want to make an ad request when a cuepoint change has been observed. To do this, an ad integration would need to do somthing like this:
+
+`player.ads.cueTextTracks.process(player, processTrack)`
```

Comment:
Here are the naming changes we came up with in our discussion just now:

process      -> processMetadataTracks*
prepareTrack -> setModeAndProcess
  setTrackMode -> setMetadataTrackMode*
  processMetadataTrack -> processMetadataTrack*